test/versioned/node-rdkafka/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "kafka-tests",
+  "targets": [{"name":"node-rdkafka","minAgentVersion":"11.18.0"}],
```

Comment:
we'll have to tweak this once we release. we just released 11.18.0 so it's something greater than that